rdtools/normalization.py

```diff
@@ -486,6 +488,8 @@ def energy_from_power(power, target_frequency=None, max_timedelta=None):
         returned for that interval. If omitted, `max_timedelta` is set
         internally to the median time delta in `power`. Ignored when `power`
         has fewer than two elements.
+    power_type : {'right_labeled', 'instantaneous'}
```

Comment:
Oh, good point. Either raise or point out that power_type is ignored for single-value input. Preference?

Reply:
I guess preference for raising.  Otherwise you get inconsistent results if you pass in a 2-element input and then only pass in the last element, both with "instantaneous" -- seems like you should get the same last value either way.